doc/developer_guide/working_practises.rst

```diff
@@ -447,6 +439,9 @@ computational cost (so that we 'fail fast'):
  3. The full test suite is run for Python versions 2.7, 3.5 and 3.8 but without
     the compilation checks.
 
+ 4. All examples in the Developer Guide are checked for correctness by
```

Comment:
This should be number 2 in the list as that's the order we give the jobs to GHA.

Reply:
OK, resorted.